sdk/appcenter-push/src/main/java/com/microsoft/appcenter/push/Push.java

```diff
@@ -430,7 +429,10 @@ private synchronized void registerPushToken() {
             onTokenRefresh(FirebaseUtils.getToken());
             AppCenterLog.info(LOG_TAG, "Firebase SDK is available, using Firebase SDK registration.");
         } catch (FirebaseUtils.FirebaseUnavailableException e) {
-            AppCenterLog.info(LOG_TAG, "Firebase SDK is not available, using built in registration. cause: " + e.getMessage());
+            AppCenterLog.warn(LOG_TAG, "Firebase SDK is not available, using built in registration. " +
+                    "Starting Android P at release date and later all versions of Android after April 2019, " +
+                    "Firebase SDK is required to use push. Please follow the migration guide at https://aka.ms/acfba.\n" +
```

Comment:
Link only tells how to remove firebase dependency.

Reply:
See @guperrot's comment above https://github.com/Microsoft/AppCenter-SDK-Android/pull/744#discussion_r198592561